source/administration/devops-command-center.rst

```diff
@@ -3,199 +3,255 @@ Mattermost Incident Management
 
 *Available in Mattermost Enterprise Edition E20, Mattermost Cloud Professional, and Mattermost Cloud Enterprise.*
 
-Incidents are situations which require an immediate response and benefit from a clearly defined process guiding towards resolution. Mattermost Incident Management allows your team to coordinate, manage, and resolve incidents from within Mattermost. 
+Incidents are situations which require an immediate response and benefit from a clearly defined process guiding toward resolution. Mattermost Incident Management equips your team to coordinate, manage, and resolve incidents from within Mattermost:
 
-Better response to an incident helps you provide more reliable services, in addition to gaining insights with incident reports and incorporating learnings with playbooks.
+* Starting an incident automatically creates a dedicated incident channel to coordinate discussion. 
+  - Use custom categories in the left-hand sidebar to streamline access to ongoing incidents.
+* Playbooks define the tasks toward resolution as your team manages the incident.
+  - Define your playbooks in advance with automated actions such as creating a Jira ticket, starting a Zoom call, or finding out who is on-call in Opsgenie.
+* Reviewing the timeline and conversation helps refine the process for future such incidents.
+  - Export the incident channel conversation to conduct a post-mortem review.
 
-- Automatically create a new channel that can be organized in the left-hand sidebar using custom categories.
-- Use playbooks to perform automated actions such as create a Jira ticket, start a Zoom call, or find out who is on-call in Opsgenie.
-- Iterate on and refine processes after each incident.
-
-When incidents are monitored, coordinated, and measured effectively, you can add transparency, maximize effectiveness, and save costs by cutting down time taken to respond to and resolve incidents.
+Delivering more reliable services requires an efficient resolution process. Use Mattermost Incident Management not only to resolve incidents more efficiently, but also to learn from past incidents and refine the process your team relies upon for success.
```

Comment:
I really like this line. Not only does MM IM give you the tools to deliver more reliable services, but help you learn from the past!